Source/ProgressionSystemRuntime/Public/Components/PSHUDComponent.h

```diff
@@ -54,8 +54,13 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSHUDComponent final : public UActorComponen
 	* Protected functions
 	********************************************************************************************* */
 protected:
+	/** Called when progression module ready
+	 * Once the save file is loaded it activates the functionality of this class */
+	UFUNCTION(BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
+	void OnInitialized(); 
+	
 	/** Subscribes to the end game state change notification on the player state. */
-	UFUNCTION()
+	UFUNCTION(BlueprintCallable, Category = "C++", meta = (BlueprintProtected))
```

Comment:
done https://github.com/h4rdmol/ProgressionSystem/pull/21/commits/14d97b4b2ec271293d8eb213cdf9bf66db7e5937

Reply:
@h4rdmol, BlueprintNativeEvent is not meta, but specifier: it should be put before BlueprintCallable, like next:
`UPROPERTY(BlueprintNativeEvent, BlueprintCallable, ...)`, otherwise it will not work for BP developers

Reply:
I got it now, thanks: https://github.com/h4rdmol/ProgressionSystem/pull/21/commits/c6ada60229141f6df9e1d5989def8584587aecf0